conf/messages

```diff
@@ -39,6 +39,16 @@ capture-submitted-vat-return.title                          = Are you currently
 capture-submitted-vat-return.heading                        = Are you currently submitting VAT Returns?
 capture-submitted-vat-return.error.message                  = Select yes if you are submitting VAT returns
 
+# VAT Application Number Page
+capture-vat-application-number.title                          = What is your VAT application number?
+capture-vat-application-number.heading                        = What is your VAT application number?
```

Comment:
The title and heading are repeated, we could just use one in both places to avoid the repeat